cookbook/case_studies/ml_training/nlp_processing/requirements.in

```diff
@@ -5,3 +5,4 @@ nltk
 matplotlib
 mpld3
 pyemd
+sklearn
```

Comment:
forgot to add sklearn - thought it was included in `requirements-common.in` but realised it was not built in image

Reply:
Can you replace `sklearn` with `scikit-learn`?

Reply:
oops my bad 😆  - thanks for spotting